docs/release-notes/17-0-0/README.md

```diff
@@ -3,24 +3,255 @@ title: OpenProject 17.0.0
+
+Existing permission configurations are automatically migrated so that **current behavior remains unchanged** unless administrators choose to restrict visibility.
+
+![Administration in OpenProject to set permissions for a global role, the permission to "View all users and groups" is checked](openproject-17-0-global-permission-view-all-users-and-groups.png)
+
+### Updated invitation flow with Primer components
+
+The user invitation dialog has been redesigned with Primer for a clearer and more consistent experience. The flow now shows only the projects where a user is allowed to invite members, improves autocompletion for users and emails, and displays errors inline instead of multi-step confirmations. This update also prepares the system for the new [visibility rules introduced in this release](#option-to-guard-the-privacy-of-users-not-working-in-the-same-project-more-strictly).
+
+screenshot
+
+### Smarter global search including type and status, improving precision in several autocompleters
+
+The global search now understands **work package type**, **status**, and **meta-status** (open/closed). This makes it much easier to find the right item in projects with many similarly named work packages.
+
+Examples:
+
+- Searching for *bug* shows only work packages of type *Bug*.
+- Searching for *bug new* shows only work packages of typ *Bug* and status *New*.
+- Searching for *open* shows only items in an open status.
+
+These improvements **also benefit several autocompleters throughout the application**, such as the `#` and `##` work package quick-link references in comments and descriptions, helping users filter large datasets more precisely.
+
+![Global search bar in OpenProject, "bug new" is typed in and two work packages of type bug and status new are displayed](openproject-17-0-search.png)
+
+### Accessibility improvements with ALT texts and improved chart colors
+
+OpenProject 17.0 continues the accessibility initiative with several enhancements:
+
+- Meaningful ALT text added to functional images (e.g., avatars, document previews).
+- Decorative images are now correctly marked to be skipped by screen readers.
+- Charts on the project overview include improved color contrast and descriptive summaries to support assistive technologies.
+- The Gantt chart is now hidden from screen readers to avoid confusing non-informational output.
+
+These improvements make OpenProject easier to navigate for users relying on screen readers or high-contrast environments.
+
+### Enhanced project attribute help texts with captions and direct editing
+
+Administrators can now **manage attribute help texts directly from each project attribute or custom field** without navigating to the separate *Attribute help texts* section. Additionally, a new **caption** field allows adding short clarifying text shown below input fields in forms to guide users more effectively.
```

Comment:
I don't know whether we should mention that displaying the caption currently only works for project attributes but not for WorkPackage custom fields?